aup.md

```diff
@@ -13,7 +13,7 @@ Violations of this Policy may result in the suspension or termination of your ac
 
 Users and Contributors must not submit, post, or share any content that:
 
-- **Illegal or Unlawful**: Violates any applicable laws, including content that promotes illegal activities, constitutes fraud, or violates intellectual property rights or other laws, regulations, and guidelines in your jurisdiction.
+- **Illegal or Unlawful**: Violates any applicable laws. Examples include (but are not limited to) content that promotes illegal activities, constitutes fraud, violates intellectual property rights, or violates other laws, regulations, or guidelines in any of the relevant jurisdictions.
```

Comment:
```suggestion
- **Illegal or Unlawful**: Violates any applicable laws. Examples include (but are not limited to) content that promotes illegal activities, constitutes fraud, violates intellectual property rights, or violates other laws, regulations, or guidelines in your jurisdiction or that of England and Wales.
```

Reply:
We may face situations where the relevant jurisdiction is not that of the developer or R-multiverse. For example, suppose a developer from Denmark uploads indecent material and a user from America downloads it. US laws might apply.

Reply:
So I would rather not try to specify or predict which specific jurisdictions are going to be relevant in all cases.

Reply:
> We may face situations where the relevant jurisdiction is not that of the developer or R-multiverse. For example, suppose a developer from Denmark uploads indecent material and a user from America downloads it. US laws might apply.

No, in this example, I don't believe US law would apply. If we're not clear then that's problematic.

Reply:
> No, in this example, I don't believe US law would apply. If we're not clear then that's problematic.

I'm having trouble understanding what would exclude US law here. In this scenario, the user finds themself in possession of material which is illegal to possess in the United States. That would violate US law. I'm not a lawyer though, so maybe I'm missing something.

Reply:
What I mean by problematic is that it leaves open the possibility that it becomes unenforceable, which is a real risk. I'm comfortable with my markup, based on my understanding, and based on the UK cyber security precedent I linked to in the issue.

Reply:
This apparently happened with Bitcoin when indecent material was part of the blockchain. Seems like that would violate laws in every country with such laws which has servers in the blockchain network.

Reply:
> > No, in this example, I don't believe US law would apply. If we're not clear then that's problematic.
> 
> I'm having trouble understanding what would exclude US law here. In this scenario, the user finds themself in possession of material which is illegal to possess in the United States. That would violate US law. I'm not a lawyer though, so maybe I'm missing something.

Sorry my post above was made at the same time you posted. My answer is, the clue is in the instruction: `Users and Contributors must not submit, post, or share any content that:` it's targeted at the content creator, not the viewer. The viewer indemnifies R-multiverse through the Terms of Use (not the AUP) if they view content that's illegal in their jurisdiction.

Reply:
> The viewer indemnifies R-multiverse through the Terms of Use (not the AUP) if they view content that's illegal in their jurisdiction.

I agree, but shouldn't our AUP make an effort to protect users in their own jurisdictions?

Reply:
Actually I just saw the clause begins with "Violates any applicable laws. Examples include (but are not limited to) ..." and the qualifier about jurisdictions is just a demonstrative example and not limiting in scope. I'm fine to merge it because the main statement is "Violates any applicable laws", whatever laws may be applicable, full stop.